dxcint/dependencies/config/cwltool.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "cwltool",
+  "languages": ["cwl"],
+  "version": "3.1.20220623174452",
```

Comment:
Will these scripts be used in the release? https://github.com/dnanexus/dxCompiler/blob/85f5deaa590125bcceea08786e598bbb5be9f15d/.github/workflows/release.yml#L68?

Reply:
Possibly. I would gradually migrate to `dxcint` for all such things - wherever possible for the sake of unified processes and documentation.